### DISCUSSION TITLE
MCC should have a Discord server

### DISCUSSION THREAD

**Discussion by Splarkszter** [ORelio/Minecraft-Console-Client#2169] [Ideas]
Would be cool to set up a discord server to hang out, share ideas more directly, announcements be broadcasted and more visible and so. 

Also to have things much more organized and that normies can be updated because GitHub is confusing for the majority of people.

**Comment by Splarkszter** [comment]
Also this could lead to the community to be expanded with potential to have more contributors.

**Comment by milutinke** [comment]
I was just thinking about it yesterday.
@ORelio What do you think?

**Comment by breadbyte** [comment]
If we get one, this could lead to faster communication within team members and issue reports.

**Reply by milutinke** [reply]
> If we get one, this could lead to faster communication within team members and issue reports.

Indeed.
Having everything neatly organized would help.
Also, a lot of people do not have Github accounts, and nowdays like everyone has a Discord account.

An idea for a layout:

**Info section**
  - A channel with git commits/releases (this would be automatically updated by a bot/webhook)
  - A channel for announcements
  - New members channel
  
**Main section**
  - General
  - FAQ
  - Help
  - Ideas/Suggestions
  - Bug report
  
**Scripting and Contributing**
 - Getting into C# and Scripting
 - Scripting help
 - Contibutors
 
 **Admin area**
  - Top Secret Channel xD

**Comment by ORelio** [comment]
I'd have very limited time to manage a Discord server or respond to questions but if other maintainers are interested I could create the server and make you (maintainers) admin so you could create the channels and moderate the server. Would that be OK with you?

**Reply by milutinke** [reply]
Fine with me

**Reply by milutinke** [reply]
Send a link when you make it.

**Reply by ORelio** [reply]
@milutinke Done! ~~https://discord.gg/removedInvite~~

**Comment by ORelio** [comment]
We now have a Discord server; see #2183
Congrats to @milutinke for configuring it in a matter of hours 👏 😁
